app/models/course_user.rb

```diff
@@ -89,6 +89,8 @@ class CourseUser < ActiveRecord::Base
       order('achievement_count DESC, last_obtained_achievement ASC')
   end)
 
+  scope :order_alphabetically, ->(direction = :asc) { order(name: direction) }
```

Comment:
I believe we have a convention for scopes that deal with sorting. Does this stick to it?